src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -3169,6 +3169,8 @@ public enum GCNotificationStatus
         public static bool operator <=(System.Guid left, System.Guid right) { throw null; }
         public static System.Guid Parse(System.ReadOnlySpan<char> input) { throw null; }
         public static System.Guid Parse(System.ReadOnlySpan<char> s, System.IFormatProvider? provider) { throw null; }
+        public static System.Guid Parse(System.ReadOnlySpan<byte> utf8Text) { throw null; }
+        public static System.Guid Parse(System.ReadOnlySpan<byte> utf8Text, System.IFormatProvider? provider) { throw null; }
```

Comment:
Format: `byte` is sorted in front of `char`, so `ROS<byte>` overloads should be first. See other types implementing `IUtf8SpanParsable`.